Preference Access SSN Matching/REPWRITERSPECS/RB.PREFACCESS.SSNMATCHING

```diff
@@ -0,0 +1,1628 @@
+**    d. Update the report with the account and PA list information
+**       along with the action taken for each of the PA records.
+**    e. Output the necessary data to the FM report to be used by Misc.
+**       Processing to update the member�s accounts.
+**
+** <PROMPTS>
+**   There are no batch prompts as the program's run-time parameters are
+**   set in a configuration Letter file.
+**
+** <INPUT>
+**   <LETTERFILE>
+**     PREFACCESS.SSNMATCHING.CFG
+**       Program configuration parameters. Instructions for setting
+**       parameters are within the Letter file.
+** <OUTPUT>
+**   <REPORTS>
+**     Pref Access SSN Update RPT
+**       Summary Report
+**     Pref Access SSN Update FM
+**       Used by Misc. Processing to update the member's accounts
+**
+** <SUPPORT FILES>
+**
+** <FM PERFORMED>
+**   Preference Access records created, updated and/or deleted
+**
+** <ADDITIONAL INFO>
+**   The program can process up to 200 alternate accounts for each
+**   home account found and up to
+** <SETUP>
+**   Update configuration Letter file as needed
+**
+** <RUNTIME>
+**   1. Run this program from batch control.
+**   2. From batch control, run Miscellaneous Processing using the report titled
+**     "Pref Access SSN Update FM" as input and verify the Miscellaneous
+**      Processing run completed successfully and results are satisfactory.
+**
+** <MOD HISTORY>
+**   <VER 1.0.0>
+**     PROGRAMMER:  TKainz Symitar
+**     DATE:        05/03/2023
+**     RELEASE:     2022.01 SP2
+**     CASE/DEFECT: N/A - Internal
+**     DETAILS:     Original Programming
+**
+*]
+
+TARGET=ACCOUNT
+
+DEFINE
+ #INCLUDE "RD.GETDATA.DEF"
+ #INCLUDE "RB.LISTEXPAND.DEF"
+ #INCLUDE "RD.OUTPUT.DEF"
+ TRUE                        = 1
+ FALSE                       = 0
+ DATENULL                    = '--/--/--'
+ BNODEBUGMODEOFF             = 0
+ BNODEBUGMODEON              = 1
+ BNOACCTTYPESMAX             = 9999
+ BNONAMETYPESMAX             = 99
+ BNOEXTLOANTYPESMAX          = 8
+ BNOCFGPARAMMAX              = 19
+ BNOERRORCODENONE            = 0
+ BNOECODECFGOPENERROR        = 510
+ BNOECODECFGREADERROR        = 511
+ BNOECODECFGDUPLICATEENTRY   = 512
+ PASTATUSUNKNOWN             = 0
+ PASTATUSVALID               = 1
+ PASTATUSNEW                 = 2
+ PASTATUSINVALID             = 3
+ PASTATUSUPDATE              = 4
+ PASTATUSIGNORE              = 5
+
+ PACOUNTMAX                  = 300
+ PAVALIDATE                  = 0
+ PADELETE                    = 1
+ PAIGNORE                    = 2
+ TARGETACCTMAX               = 200
+
+ NO                          = 0
+ YES                         = 1
+ EXISTING                    = 2
+ NEW                         = 3
+
+ ACCOUNTADDED                = NUMBER
+ ACCOUNTCOUNT                = NUMBER
+ ACCOUNTLIMIT                = NUMBER
+ ACCTCLOSEDATE               = DATE
+ ACCTNAMEFOUND               = NUMBER
+ BASEACCOUNT                 = CHARACTER
+ BNOCFGFILEERROR             = CHARACTER
```

Comment:
BNOCFGFILEERROR is never used in the code. BNOCFGFILEFERROR is used instead.
Is BNOCFGFILEERROR for future use?